src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java

```diff
@@ -515,6 +520,16 @@ private SELF withDebug(int hostPort, boolean suspend) {
         return self();
     }
 
+    /** Disable default bootstrapping of the keycloak admin. Useful when realms are imported. */
+    public SELF withoutBootstrapAdmin() {
```

Comment:
Perhaps the method should be named `withBootstrapAdminDisabled()`!? WDYT?
IMHO it's easier to read, as "without" is too similar with "with" and it's not clear enough that with this setting something will be disabled.

Reply:
Amended